src/main/java/duke/command/MarkCommand.java

```diff
@@ -0,0 +1,50 @@
+package duke.command;
+
+import duke.exception.InvalidArgumentException;
+import duke.exception.OutOfBoundsException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Messages;
+import duke.ui.Ui;
+
+import java.io.IOException;
+
+public class MarkCommand extends Command {
+
+    private String[] inputWords;
+
+    public MarkCommand(String[] inputWords) {
+        this.inputWords = inputWords;
+    }
+
+    /**
+     * Marks the task as done.
+     *
+     * @param tasks TaskList that command is executed on.
+     *
+     * @param ui User interface that interacts with the user.
+     *
```

Comment:
Based on the javadocs guide in module website, I think there shouldn't be new line between @param.

Reply:
i agree with you! Thank you, will fix that!